### PR TITLE
Changed semantics for instance search filter searchString

### PR DESCRIPTION
**Pull request by HenningNormann**
## Description
Changed semantics for instance search filter searchString so that the search string is used both in app titles and in presentation fields.

## Related Issue(s)
- https://github.com/Altinn/altinn-support-private/issues/2850

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Storage/Migration/v0.08/01-functions-and-procedures.sql [hunk 1]
```diff
@@ -0,0 +1,539 @@
+-- This script is autogenerated from the tool DbTools. Do not edit manually.
```
Comment (acn-sbuad) [inline src/Storage/Migration/v0.08/01-functions-and-procedures.sql:1]: `readinstancefromquery` can be dropped

**Comment by acn-sbuad** [comment]
Can we fix the failing condition coverage requirement?

File: src/Storage/Migration/v0.08/01-functions-and-procedures.sql (hunk 1 above)
Comment (HenningNormann) [reply src/Storage/Migration/v0.08/01-functions-and-procedures.sql:1]: Done

**Comment by HenningNormann** [comment]
> Can we fix the failing condition coverage requirement?
I can't see that the new code as low coverage as reported.

**Comment by SandGrainOne** [comment]
> Can we fix the failing condition coverage requirement? I can't see that the new code as low coverage as reported.

It's complaining about condition coverage. Not code coverage in general. There are a few checks where one of the outcomes are untested. The coverage overall is okay.

**Review by acn-sbuad — APPROVED** [review]
🎈

**Pull request merged**